app/models/null_device.rb

```diff
@@ -0,0 +1,13 @@
+class NullDevice < Device
+  def save
+    no(__method__)
+  end
+
+  def save!
+    no(__method__)
+  end
+
+  def no(method)
```

Comment:
Unused method argument - `method`. If it's necessary, use `_` or `_method` as an argument name to indicate that it won't be used. You can also write as `no(*)` if you want the method to accept any arguments but don't care about them.